src/Sylius/Bundle/OrderBundle/Doctrine/ORM/OrderRepository.php

```diff
@@ -56,6 +56,22 @@ public function findLatest($count)
     /**
      * {@inheritdoc}
      */
+    public function createCartsPaginator(array $criteria = [], array $sorting = [])
```

Comment:
Should be replaced with ``createCartListQueryBuilder`` method and used with grids - it will take care of filtering and sorting.